Gemfile.lock

```diff
@@ -249,25 +249,25 @@ GEM
     mini_magick (4.10.1)
     mini_mime (1.0.2)
     mini_portile2 (2.4.0)
-    minitest (5.14.1)
+    minitest (5.14.2)
     msgpack (1.3.3)
     net-scp (3.0.0)
       net-ssh (>= 2.6.5, < 7.0.0)
-    net-ssh (6.0.2)
+    net-ssh (6.1.0)
```

Comment:
### [net-ssh](https://github.com/net-ssh/net-ssh)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 6.0.2 → 6.1.0

[_[change-log](https://github.com/net-ssh/net-ssh/blob/master/CHANGES.txt), ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **17** commits. See the full changes on [the compare page](https://github.com/net-ssh/net-ssh/compare/v6.0.2...v6.1.0).

These are the first 10 commits:
- (c457c06) [Support algorithm subtraction syntax from ssh_config](https://github.com/net-ssh/net-ssh/commit/c457c06394c8e182c9b213fa7a2738c37bd47f7d)
- (84f1705) [Minor improvements to README.md.](https://github.com/net-ssh/net-ssh/commit/84f17052295d5cae1ad673a51752f351815760c2)
- (c3536f6) [Minor wording improvement.](https://github.com/net-ssh/net-ssh/commit/c3536f6ce536e5b1672021a3c8b2ac252f94d749)
- (9bed3c7) [Merge pull request #751 from maxfierke/mf-support_algo_subtr…](https://github.com/net-ssh/net-ssh/commit/9bed3c7f210ac5de54e6b787cce8498f10ce5c79)
- (e40df16) [Add test to reproduce corrupted Mac issue](https://github.com/net-ssh/net-ssh/commit/e40df165d5f34fe4eaebe3bdb3b4a14499efb737)
- (ebc6e4b) [Fix corrupted hmac detected](https://github.com/net-ssh/net-ssh/commit/ebc6e4b1cb3e14e8aead1adc464d223324eb0028)
- (104350b) [Fixed test_alg](https://github.com/net-ssh/net-ssh/commit/104350b94cf0a27e5c0b786dd8e90944273f64aa)
- (0294c53) [Update README.md](https://github.com/net-ssh/net-ssh/commit/0294c5383c1f7a6560fc4d0517b94e43a6b7b903)
- (39d70af) [Merge pull request #759 from mfazekas/fix-corrup-hmac-issue](https://github.com/net-ssh/net-ssh/commit/39d70af82e0a9ab0e523ef34af1460fa666ff610)
- (996b6c8) [Merge pull request #760 from net-ssh/fwininger-patch-1](https://github.com/net-ssh/net-ssh/commit/996b6c81fc5c42e015d3e3893a9c2acab635d264)


</details>